scrum-of-scrums/readme.md

```diff
@@ -1,23 +1,33 @@
 # Scrum of Scrums
 
-] This content is under construction. For now, for those who are unfamiliar: [Scrum of Scrums](https://www.agilealliance.org/glossary/scrum-of-scrums)
+Scrum or scrums is a technique used to scale Scrum to a larger group. In Scrum we consider a team being too big when going over 10-12 individuals. This should be decided on a case by case basis. If the project is set up in multiple workstreams that contain a fixed group of people and a common [stand-up](../stand-ups/readme.md) meeting is slowing down productivity: scrum of scrums should be considered. You would identify the different sub teams that would almost act as separate scrum team.

```

Comment:
suggestion to complete with: "... to a larger group working under the same project goal." 
Consider also that scrum of scrums may facilitate the communication and help to convey interdependencies on different workstreams.